Px.Utils/Validation/IValidationFeedback.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace PxUtils.Validation
+{
+    /// <summary>
+    /// Defines a common interface for validation feedback. Validation feedback represents the result of a validation operation.
+    /// </summary>
+    public interface IValidationFeedback
```

Comment:
This could also be an abstract class.

Reply:
Made this into one struct that all feedback item objects are created from